vendor/modules.txt

```diff
@@ -0,0 +1,5 @@
+# github.com/cenkalti/hub v1.0.1-0.20160527103212-11382a9960d3
```

Comment:
I think we should remove vendor directory from the repo, when using go mod. What do you think?

Reply:
yes, thanks i missed this. vendor dir needed only in final product that used this library